mmv1/products/compute/api.yaml

```diff
@@ -7701,7 +7701,6 @@ objects:
         description: |
           The regional or multi-regional Cloud Storage bucket location where the machine image is stored.
         item_type: Api::Type::String
-        output: true
```

Comment:
Hey @Yannic,

It turns out we can't just remove this attribute. For fields that were API value only, there may be situations where the user has not specified the field, but it is returned by the API anyway. We need to give it the following line in the `compute/terraform.yaml`:
```
      storageLocations: !ruby/object:Overrides::Terraform::PropertyOverride
        default_from_api: true
```
Which makes it default to the API value if the user has not explicitly specified it in their configuration.

Reply:
thanks, done

Reply:
wait, I misread you message. Let me update the other file